.github/workflows/tests-ci.yml

```diff
@@ -34,30 +38,34 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
         with:
           submodules: true
+    
+      - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d
+        with:
+          version: 8
 
       - name: Set up Node.js
         uses: actions/setup-node@5ef044f9d09786428e6e895be6be17937becee3a #v4.0.0
         with:
           node-version: 18
           registry-url: https://registry.npmjs.org/
-          cache: "npm"
+          cache: "pnpm"
```

Comment:
Same

Reply:
Addressed in [0e8d453](https://github.com/TBD54566975/web5-js/pull/419/commits/0e8d453240e996dc85fca63441c5065ede97b82c)